### PR TITLE
main/f2fs-tools: upgrade to 1.13.0

### PR DESCRIPTION
**Pull request by jjsearle**

### CODE REVIEW

File: main/f2fs-tools/APKBUILD [hunk 1]
```diff
@@ -1,29 +1,23 @@
...
-install=""
 subpackages="$pkgname-doc $pkgname-dev $pkgname-libs"
-source="https://git.kernel.org/cgit/linux/kernel/git/jaegeuk/f2fs-tools.git/snapshot/$pkgname-$pkgver.tar.gz
+source="https://git.kernel.org/cgit/linux/kernel/git/jaegeuk/f2fs-tools.git/snapshot/f2fs-tools-$pkgver.tar.gz
 	"
-builddir="$srcdir/$pkgname-$pkgver"
 
 prepare() {
 	default_prepare
-	cd "$builddir"
```
Comment (maxice8) [inline main/f2fs-tools/APKBUILD]: packages in main/ don't lose this.

https://gitlab.alpinelinux.org/Leo/atools/blob/master/apkbuild-lint#L298

Comment (jjsearle) [reply main/f2fs-tools/APKBUILD]: Not sure I understand the rational behind this but I've added them back in.

Comment (maxice8) [reply main/f2fs-tools/APKBUILD]: main/ is 2 year support which has versions of  Alpine that  do not support it.

Comment (jjsearle) [reply main/f2fs-tools/APKBUILD]: Thanks for the explanation

**Comment by maxice8** [comment]
28f45698a58920de9f35af40ae0a964c37ce9fe9

**Pull request closed without merging**